nucypher/policy/models.py

```diff
@@ -255,6 +255,7 @@ def enact(self, network_middleware, publish=True) -> dict:
         else:  # ...After *all* the policies are enacted
             # Create Alice's revocation kit
             self.revocation_kit = RevocationKit(self, self.alice.stamp)
+            self.alice.active_policies[bytes(self.public_key).hex()] = self
```

Comment:
Mostly because we're accessing the policies from the character control API.

Obviously, this storage is problematic mostly because one can immediately imagine having multiple policies per pubkey. I made a point in your other comment that this will all, hopefully, be answered in another PR that builds a much better solution.

Reply:
Fair enough! Do you have this controlled or would it be good to file an issue?